### PR TITLE
add more information in Readme file

### PR DESCRIPTION
**Pull request by camilog**
A few improvements to README file.
Please check it if it's ok (understandable).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,54 @@ $ source env/bin/activate
...
+}
+```
+
+Here's an explanation of what means each of the values above:
+```<redis_server_ip_address>```: at the moment of running Fievel, a Redis server is launched. You must put here the IP Address of that Redis server.
+```<redis_server_port>```: port where the Redis server is running.
+
+```<url_sse_events_path>```: a query in Gopher will be at: gopher_address:8080/<url_sse_events_path>/<active_event_processor>
+
+```<active_event_processor>```: currently Gopher supports three Event Processor, and you can set which of the three (or all of them) will be active for this instance:
```
Comment (Zeeerooo) [inline README.md]: Maybe you can explain better how to use the tree processor at the same time

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 RaTA DNS aggregator module. Service that process DNS packets information and sends it to a HTML5 SSE.
 
-# Developer info
+# Installation
```
Comment (fmontoto) [inline README.md]: Why did you change it to installation if you're giving developent instructions?

File: README.md [hunk 3]
```diff
@@ -13,9 +13,54 @@ $ source env/bin/activate
 (env) $ pip install -r requirements.txt
 ```
 
-## Testing
+# Configuration File
```
Comment (fmontoto) [inline README.md]: Maybe a simple example to run the software ASAP

File: README.md [hunk 4]
```diff
@@ -13,9 +13,54 @@ $ source env/bin/activate
...
+Here's an explanation of what means each of the values above:
+```<redis_server_ip_address>```: at the moment of running Fievel, a Redis server is launched. You must put here the IP Address of that Redis server.
+```<redis_server_port>```: port where the Redis server is running.
+
+```<url_sse_events_path>```: a query in Gopher will be at: gopher_address:8080/<url_sse_events_path>/<active_event_processor>
+
+```<active_event_processor>```: currently Gopher supports three Event Processor, and you can set which of the three (or all of them) will be active for this instance:
+1. "server_data": delivers QueriesPerSecond and AnswersPerSecond of each DNS server that is being monitored.
+2. "queries_by_qtype": delivers information about the top queries made by each of the different qtypes that exists.
+3. "top_qnames_with_ip":  delivers information about the top domains being queried, also giving information about the IP addresses that are doing all those queries.
```
Comment (Zeeerooo) [inline README.md]: Maybe is better use "... are querying ..."

File: README.md [hunk 5]
```diff
@@ -13,9 +13,54 @@ $ source env/bin/activate
...
+    "<active_event_processor_3>"
+  ],
+  "<active_event_processor_1>": {
+    "times": [<time_window_length>],
+    "output_limit": <event_processor_output_quantity>
+  }
+}
+```
+
+Here's an explanation of what means each of the values above:
```
Comment (fmontoto) [inline README.md]: I don't like the sentence :( 
Maybe "Meaning of the values used above".  ?

File: README.md [hunk 6]
```diff
@@ -13,9 +13,54 @@ $ source env/bin/activate
...
+```<redis_server_ip_address>```: at the moment of running Fievel, a Redis server is launched. You must put here the IP Address of that Redis server.
```
Comment (fmontoto) [inline README.md]: Too verbose.
IP Address of the Redis server running at fievel?

File: README.md [hunk 7]
```diff
@@ -13,9 +13,54 @@ $ source env/bin/activate
...
+```<url_sse_events_path>```: a query in Gopher will be at: gopher_address:8080/<url_sse_events_path>/<active_event_processor>
+
+```<active_event_processor>```: currently Gopher supports three Event Processor, and you can set which of the three (or all of them) will be active for this instance:
+1. "server_data": delivers QueriesPerSecond and AnswersPerSecond of each DNS server that is being monitored.
+2. "queries_by_qtype": delivers information about the top queries made by each of the different qtypes that exists.
+3. "top_qnames_with_ip":  delivers information about the top domains being queried, also giving information about the IP addresses that are doing all those queries.
+For more information about each Event Processor please refer to the [wiki page](https://github.com/niclabs/ratadns-gopher/wiki).
+
+```[<time_window_length>]```: array containing all the time windows length (in minutes) that will be managed in this instance.
+```<event_processor_output_quantity>```: depending on each Event Processor, this number will set how many will be considered as the "top" in the ranking.
```
Comment (fmontoto) [inline README.md]: I don't understand this explanation

File: README.md [hunk 8]
```diff
@@ -13,9 +13,54 @@ $ source env/bin/activate
...
+
+# Running Gopher
+To start running Gopher you must have an at least one instance of [RataDNS-Fievel](https://www.github.com/niclabs/ratadns-fievel) running too.
```
Comment (fmontoto) [inline README.md]: remove running. 'To start gopher' is enough

Comment (Zeeerooo) [inline README.md]: remove "an"

**Comment by fmontoto** [comment]
I think it's unnecessary to talk about virtual environments in this README, as it's not intended to provide python good practices or something like that.

File: README.md (hunk 1 above)
Comment (camilog) [inline README.md]: I was gonna explain more details on the wiki page

File: README.md (hunk 3 above)
Comment (camilog) [inline README.md]: After this section there's a "Running Gopher"

File: README.md (hunk 6 above)
Comment (camilog) [inline README.md]: I didn't like it too much when I wrote it

File: README.md (hunk 7 above)
Comment (camilog) [inline README.md]: I will improve it in the next commit

File: README.md (hunk 2 above)
Comment (camilog) [inline README.md]: I thought they were Instructions for Installation (for "common users") more than information for people that wants to help in the development of the application. Maybe I will add more detailed information abut developing in the wiki page.

**Comment by camilog** [comment]
I think the same as you @fmontoto about virtualenv, but this comes from previous developers. I will remove it.

**Comment by fmontoto** [comment]
LGTM

**Comment by Zeeerooo** [comment]
LGTM

**Comment by storress** [comment]
LGTM
